src/App.vue

```diff
@@ -11,9 +11,7 @@
         >
           <template #header>
             <template v-if="download.source.type === 'branch'">
-              <ElTag effect="dark" type="primary" disableTransitions
-                >Branch</ElTag
-              >
+              <ElTag type="primary" disableTransitions>Branch</ElTag>
```

Comment:
```suggestion
              <ElTag effect="dark" type="primary" disableTransitions
                >Branch</ElTag
              >
```
effect=darkの方がコントラスト的にも見やすいかも？
![image](https://github.com/user-attachments/assets/18ab4363-3258-419b-94df-5ede9b32f6b1)

Reply:
情報の大切さ的に、これがbranchなのかとエディタボタンだったらボタンのが重要そうで、それに応じた目立ち度に従う場合は薄いほうが合ってるかも。

Reply:
あ〜なるほど、ならこれで良さそう．